Formula/terragrunt.rb

```diff
@@ -25,5 +25,10 @@ def install
 
   test do
     assert_match version.to_s, shell_output("#{bin}/terragrunt --version")
+
+    # Generate an empty plan just to ensure Terraform works
+    touch "#{testpath}/terragrunt.hcl"
+    touch "#{testpath}/main.tf"
+    system "#{bin}/terragrunt plan"
```

Comment:
I just tried running `terragrunt` using `terraform` as a test dependency instead of `tfenv` and it prints the following output:
```
Initializing the backend...

Initializing provider plugins...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.
```
When I try with `tfenv` instead I get an error.  So think we can make this change
```suggestion
    system "#{bin}/terragrunt plan"
    assert_match "Terraform has been successfully initialized!", Utils.safe_popen_read(bin/"terragrunt", "plan")
```

Reply:
This change makes assumptions about the text output of a different program (terraform) that terragrunt has no control over. However terraform does exist with a success or failure return code, and terragrant does track and return that same success or failure return code. Therefore I believe that testing the return code of the command is the safer thing to do, which does not depend on textual output from Terraform. Also, the language of the output is affected by localization settings whereas return codes are not.

Reply:
We don't really have to worry about the localization affecting the output of the test because our main concern is that the test will pass in CI, which always uses an English locale.  We probably have thousands of tests that would fail with non-English locales.  

I also think it's fine if the text isn't directly controlled by `terragrunt`.  I believe we also have other test like this already, and it's helpful for other maintainers to see that the test is actually outputting a string indicating success than just looking for return 0.  If the text output of `terraform` eventually changes and the test fails because of this, it is trivial to fix because the error will print the new text so we know what needs to be changed.

Reply:
🤷‍♀️ you're the boss. I've made the change.  I just find it baffling to test a different program entirely, and to risk a test failure based on changes to a different program, while not testing very important return-code passthrough 🤷‍♀️

Reply:
The return code actually is tested as well.  If a value is not specified it defaults to expecting 0 and will fail if the return code is anything else.  Depending on another program may seem unusual but we have many other cases where this happens, and it will be more helpful for maintainers to see it producing output beyond just a return code.